### PR TITLE
[REVIEW] Correct inconsistencies in README and CONTRIBUTING docs

### PR DESCRIPTION
**Pull request by robertmaynard**
This PR corrects a couple of inconsistencies in the documentation.

### CODE REVIEW

**Review by kkraus14 — APPROVED** [review]

**Comment by kkraus14** [comment]
@gpucibot merge

File: README.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ $ source activate cudf_dev
 ```
 
 - Build and install `librmm` using cmake & make. CMake depends on the `nvcc` executable being on
-  your path or defined in `$CUDACXX`.
+  your path or defined in `CUDACXX` environment variable.
```
Comment (robertmaynard) [inline README.md:91]: @kkraus14 I know see that this doc changes are found in other RAPIDS project. I now think I should drop this change, so the docs are consistent across all projects. 

Thoughts?

Comment (kkraus14) [reply README.md:91]: We should just fix them across the other projects as well moving forward.

**Comment by kkraus14** [comment]
@gpucibot merge

**Pull request merged**